### PR TITLE
Added media queries on users table

### PR DESCRIPTION
**Pull request by LouiaR**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "global": "^4.3.2",
```
Comment (kkarimi) [inline frontend/package.json]: @LouiaR What is this used for?

Comment (LouiaR) [reply frontend/package.json]: @kkarimi Sorry I think I got it when I was trying to install yarn

Comment (kkarimi) [reply frontend/package.json]: @LouiaR that's ok! Please remove it from here

Comment (LouiaR) [reply frontend/package.json]: Done

**Pull request merged**